components/nexus/statefulset.yaml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
When Xin originally installed Nexus on our cluster, he used Deployment instead of StatefulSet. What was the reason for this change here?
https://github.com/redhat-appstudio/rhtap-utils/tree/main/nexus

Reply:
Originally had deploy.  I tested updating the release, on the rollout of new release it would hang, pvc in use. This handles the pvc and update version successfully deployed.